vendor/strega/smart-valve-codec.yaml

```
@@ -0,0 +1,52 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: smart-valve.js
+  # Examples (optional)
+  #examples:
+  # - description: Periodic Uplink
+  #  input:
+  #   fPort: 4
+  #  bytes: [333138383223633D77DB]
```

@surendar94 what I mean is:

```suggestion
  #  bytes: [0x33, 0x31, 0x38, 0x38, 0x32, 0x23, 0x63, 0x3D, 0x77, 0xDB]
```

---

@johanstokking, Could you merge this PR? Surendar says that they will add example if required later.